docs/conf.py

```diff
@@ -91,7 +91,10 @@
 # Add and use Pylons theme
 html_theme = 'pylons'
 html_theme_path = pylons_sphinx_themes.get_html_themes_path()
-html_theme_options = dict(github_url='http://github.com/Pylons/waitress')
+html_theme_options = dict(
+    github_url='http://github.com/Pylons/waitress',
+    canonical_url='http://docs.pylonsproject.org/projects/waitress/en/latest/'
```

Comment:
Yes, it is a separate package:

https://github.com/Pylons/pylons-sphinx-themes/blob/master/pylons_sphinx_themes/pylons/layout.html#L14

As far as the link in the source, that's odd. From [RTD docs](http://docs.readthedocs.io/en/latest/canonical.html?highlight=canonical#enabling)

> You can set the canonical URL for your project in the Project Admin page. Check your Domains tab for the domains that we know about.

But there is no domain listed under the Waitress RTD settings.

I have no idea why the canonical URL still shows. My guess is the RTD builds have been failing for many months, so it could have been a remnant. I'll doublecheck it once we have a successful build on RTD.